feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/SearchScreenPresenter.kt

```diff
@@ -108,5 +119,6 @@ fun searchScreenPresenter(
             availableLanguages = listOf(Lang.JAPANESE, Lang.ENGLISH),
         ),
         hasSearchCriteria = hasSearchCriteria,
+        bookmarks = selectedBookmarks,
```

Comment:
Currently, selectedBookmarks is being held as a local state with rememberRetained. However, since timetable.bookmarks is already kept up-to-date via favoriteTimetableIdsSubscriptionKey, this state management seems redundant.
I propose changing the implementation to directly pass timetable.bookmarks instead of holding selectedBookmarks. What are your thoughts?